src/buildHTML.js

```diff
@@ -1473,6 +1501,230 @@ groupTypes.accent = function(group, options) {
+        if (group.value.isOver) {
+            noteShift = braceShift - braceBody.height - 4 * ruleWidth;
+        } else {
+            noteShift = braceShift + noteReset.height + 4 * ruleWidth;
+        }
+
+        const vSpan = makeSpan(["mord",
+            (group.value.isOver ? "mover" : "munder")],
+            [vlist], options);
+
+        vlist = buildCommon.makeVList([
+            {type: "elem", elem: vSpan, shift: 0},
+            {type: "elem", elem: noteReset, shift: noteShift},
+        ], "individualShift", null, options);
+    }
+
+    return makeSpan(["mord", (group.value.isOver ? "mover" : "munder")],
+        [vlist], options);
+};
+
+groupTypes.accentunder = function(group, options) {
+    // Treat under accents much like underlines.
+    const innerGroup = buildGroup(group.value.body, options);
+
+    const ruleWidth = fontMetrics.metrics.defaultRuleThickness /
+        options.style.sizeMultiplier;
+
+    const accentBody = stretchy.svgSpan(group, options);
+    const accentShift = innerGroup.depth + 2 * ruleWidth + accentBody.height;
+
+    // Generate the vlist, with the appropriate kerns
+    const vlist = buildCommon.makeVList([
+        {type: "elem", elem: innerGroup, shift: 0},
+        {type: "elem", elem: accentBody, shift: accentShift},
+    ], "individualShift", innerGroup.height, options);
+
+    return makeSpan(["mord", "accentunder"], [vlist], options);
+};
+
+groupTypes.strikeThru = function(group, options) {
+    const innerGroup = buildGroup(group.value.body, options);
+    const style = options.style;
+
+    if (group.value.label === "\\sout") {
+        const line = makeSpan(
+            [style.reset(), Style.TEXT.cls(), "stretchy", "sout"]);
+        line.height = fontMetrics.metrics.defaultRuleThickness /
+            style.sizeMultiplier;
+        line.maxFontSize = 1.0;
+
+        const lineShift = -0.5 * style.metrics.xHeight;
+        const vlist = buildCommon.makeVList([
+            {type: "elem", elem: innerGroup, shift: 0},
+            {type: "elem", elem: line, shift: lineShift},
+        ], "individualShift", null, options);
+        return makeSpan(["mord"], [vlist], options);
+
+    } else {
+        //  \cancel, \bcancel, \xcancel
+        let node = makeSpan([], [innerGroup], options);
+
+        // buildCommon.makeVList does not work well with a tall image span.
+        // So work out vertical placement here.
+
+        // Create a strut with which to enforce a span height.
+        const strut = makeSpan([style.reset(), "strut"], [], options);
+        const totalHeight = innerGroup.depth + innerGroup.height + 0.4;
+        strut.style.height = totalHeight + "em";
+        strut.style.verticalAlign = -(innerGroup.depth + 0.2) + "em";
+
+        node = makeSpan(["inner-wrap"], [node, strut], options);
+
+        // Create a span, strikeThru, with the background-image.
+        const strikeThru = stretchy.strikeSpan(group.value.label.substr(1),
+            options);
```

Comment:
I'm not removing the first letter. I'm removing the opening `\` character. As you note elsewhere, it may have been a good idea to strip off that `\` in `functions.js`.